src/components/Autocomplete/Autocomplete.module.scss

```
@@ -0,0 +1,283 @@
+@import '~@anyvision/style-guide';
+
+%noSpacingStyle {
+  border: 0;
+  margin: 0;
+  outline: 0;
+  padding: 0;
+}
+
+%hiddenTextOverflow {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+$default-selected-height: av-size(sz-56);
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: av-size(sz-320);
+  box-sizing: border-box;
+  position: relative;
+  height: fit-content;
+
+  &.isDisabled {
+    opacity: av-opacity(disabled);
+  }
+
+  > *,
+  *::after,
+  *::before {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  .selectedContainer {
+    cursor: pointer;
+    outline: none;
+    position: relative;
+    border-radius: av-radius(tidy);
+    padding: 0 av-space() 2px;
+    height: $default-selected-height;
+    min-height: $default-selected-height;
```

Think you can get rid of height 
```suggestion
    min-height: $default-selected-height;
```